package.json

```diff
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^2.7.0",
+    "@typescript-eslint/parser": "^2.7.0",
```

Comment:
I needed this for eslint to work. I used npm maybe you are using yarn

Reply:
yes I was using yarn